### PR TITLE
Make sure authz endpoints are documented in openapi spec

### LINKED ISSUE #25259: OpenAPI spec doesn't match the admin API
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/api

### Describe the bug

Continuation of #24960 with another difference between the Admin API and the OpenAPI spec:

- None of the `/admin/realms/{realm}/clients/{client-uuid}/authz`-endpoints are currently documented in the OpenAPI spec.

### Version

23.0.0

### Expected behavior

The generated OpenAPI spec should match the admin API.

### Actual behavior

The generated OpenAPI spec doesn't match the admin API.

### How to Reproduce?

- Generate the OpenAPI spec using mvn -pl services -am -DskipTests -P jboss-release
- Check the generated OpenAPI spec in services/target/apidocs-rest/swagger/apidocs/openapi.json

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by 999eagle**
Fixes: #25259

### CODE REVIEW

File: services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java [hunk 1]
```diff
@@ -90,9 +94,11 @@ public ResourceServer create(boolean newClient) {
         return resourceServer;
     }
 
+    @Path("/update")
```
Comment (ahus1) [inline services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java:97]: It seems to me that this line was added by mistake. Once I remove it, the tests start to pass.

```suggestion
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
I had a look and I think by removing one line the tests could pass. See below...

File: services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java (hunk 1 above)
Comment (999eagle) [reply services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java]: Oh, good catch! Yeah I think this fixes the test suite, thanks :)

**Review by ghost — COMMENTED** [review]
Unreported flaky test detected, please review

**Review by ghost — COMMENTED** [review]
Unreported flaky test detected, please review

**Review by ahus1 — APPROVED** [review]
Thank you for improving the OpenAPI spec again!

**Pull request merged**